render/jpql/src/testFixtures/kotlin/com/linecorp/kotlinjdsl/render/jpql/entity/employee/Employee.kt

```diff
@@ -7,4 +7,5 @@ abstract class Employee(
     val phone: String,
     val address: EmployeeAddress,
     val departments: MutableSet<EmployeeDepartment>,
+    val age: Int,
```

Comment:
I think it would be easier to read if the age field was after the nickname, as it would group similar fields together.

Reply:
Reflected now.